instagram-app/src/components/PostContainer/Post.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import './Post.css'
+import CommentSection from '../CommentSection/CommentSection'
+import {PostActions} from './PostActions';
+import styled from 'styled-components';
+
+const ListItem = styled.li`
+    width: 100%;
+    padding: 0%;
+    background: white;
+    border: .1rem solid #e6e6e6;
+    border-radius: .3rem;
+    margin: 0;
+    margin-bottom: 6rem;
+`;
+
+class Posts extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            likes: props.post.likes
+        };
+    }
+    incrementLike = () => {
+        let likes = this.state.likes + 1;
```

Comment:
We can make this much better by not just adding +1 to likes each time the heart is clicked. 
Add a boolean state value to stated and set it to false. so liked: false
And then when you are setting state in Increment like: 
likes:     this.state.liked === false ? this.state.likes += 1 : this.state.likes -= 1
liked: !this.state.liked